.github/ISSUE_TEMPLATE/Update A-enviar-software-para-revision.md

```diff
@@ -0,0 +1,105 @@
+---
+name: Enviar software para revisión
+about: Usa esta plantilla para enviar un software a revisar en Español (Experimental)
+---
+
+
+Autor Encargado: nombre (@github_handle)  
```

Comment:
This opens the conversation about gender-biased language. I think the project "R para Ciencia de Datos" decided to use either gender neutral rephrasing or go with the female version, e.g. `Autora encargada: nombre (@github_handle)`. 

The scope of this comment is this entire file, and beyond.

Reply:
Yes I agree, either gender-neutral (my preferred solution) or female version.

Reply:
Changed to "Persona Encargada".

Reply:
I will change it to _Persona Encargada_